leo/core/leoGlobals.py

```diff
@@ -3308,6 +3308,26 @@ def readlineForceUnixNewline(f: IO, fileName: Optional[str] = None) -> str:
     if len(s) >= 2 and s[-2] == "\r" and s[-1] == "\n":
         s = s[0:-2] + "\n"
     return s
+#@+node:ekr.20250615134309.1: *3* g.relativeDirectory
+def relativeDirectory(baseDir: str, path: str) -> str:
+    """
+    Return the path relative to the base directory.
+    Return path if baseDir is not an ancestor of path."""
+    if not baseDir:
+        # Likely an unsaved outline.
```

Comment:
Just one last small thing, For users to be made aware of this detail on the spot. I think I would have printed this when _not baseDir:_

`g.es_print_unique_message('Imported using absolute path (unsaved outline). Save the outline to enable relative paths.')
`

Reply:
Thanks for this. Please see b423981